include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -73,6 +75,14 @@ struct gap
     //! internal value
```

Comment:
Please use `\privatesection` and `\publicsection` to make the internal stuff invisible to the documentation